app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -547,7 +547,15 @@ interface MetricController {
                     isDataTelemetryEnabled,
                     isMarketingDataTelemetryEnabled
                 )
-            } else DebugMetricController()
+            } else {
+                // todo switch it back to normal
+                ReleaseMetricController(
```

Comment:
No change needed: for future reference, I usually change this one line to enable telemetry for debug builds: https://searchfox.org/mozilla-mobile/rev/58d77b139e5e992a204a0c6b1f40120344f4820d/fenix/app/build.gradle#342

Reply:
Thanks!